CBLClient/Apps/CBLTestServer-Dotnet/TestServer/ValueSerializer.cs

```diff
@@ -117,7 +117,8 @@ internal static object DeserializeValue(String value)
             }
             else if (value.StartsWith("\"") && value.EndsWith("\""))
             {
-                return value.Replace("\"", String.Empty);
+                string st = value.Remove(0, 1);
+                return st.Remove(st.LastIndexOf("\""));
```

Comment:
should line 121 be return st.Remove(st.LastIndexOf("\""), 1) ? say you have a string value \"couchbase\", st will be "couchbase\", and line 121 will give you "couchbase, the last quote is completed removed, not only the back slash \

Reply:
Like python if just add """ I am seeing errors , so we need to keep the back slash.   Goal here is to remove the cotes around the string . Line 120 removes the starting cote and 121 remove the ending cote. I am use the trim methods, but it will remove all the ending cotes . We want to remove only one ending cote, hence I needed to go with this approach

Reply:
Added the swift code for this change.